fix update-release-notes when changing multiple packs (#1765)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

# 1.5.5

> * Fixed an issue in the **update-release-notes** command, where running the command when making changes in multiple packs resulted in no updates at all.

Reviewer: ```suggestion
* Fixed an issue in the **update-release-notes** command, which did not work when changes were made in multiple packs.
```
